Cargo.toml

```
@@ -58,6 +58,8 @@ rand = "0.8.5"
 # Used to hash passwords
 rust-argon2 = { git = "https://github.com/sru-systems/rust-argon2", rev = "e6cb5bf99643e565f4f0d103960d655dac9f3097" }
 reqwest = { version = "0.11.22", default-features = false, features = ["rustls-tls-native-roots", "socks"] }
+# Used to validate hostnames, already included in reqwest however we need access to it
+url  = {version = "^2", features = ["serde"]}
```

Nitpick I know but can we fix the spacing here

```suggestion
url = { version = "^2", features = ["serde"] }
```